src/auth/user-import-builder.ts

```diff
@@ -350,7 +355,21 @@ export function convertMultiFactorInfoToServerFormat(multiFactorInfo: UpdateMult
       }
     }
     return authFactorInfo;
-  } else {
+  } else if (isTotpFactor(multiFactorInfo)) {
```

Comment:
nit: since you have a return in the previous if block, this can also be if instead of else if.